### DISCUSSION TITLE
Any plans on integrating Local RAG Integration like https://github.com/open-webui/open-webui

### DISCUSSION THREAD

**Discussion by gnerativ** [mckaywrigley/chatbot-ui#1720] [Ideas]
As mentioned, any plans in integrating a local RAG integrations like https://github.com/open-webui/open-webui ?

Have a separate tab on nav bar for chat with documents and documents can be either preloaded or loaded through UI.

**Comment by raul-bermejo** [comment]
great idea @gnerativ, this could definitely be a feature request :)
